quartz_solar_forecast/inverters/enphase.py

```diff
@@ -132,8 +131,23 @@ def get_enphase_data(enphase_system_id: str) -> float:
 
     # Convert the decoded data into JSON format
     data_json = json.loads(decoded_data)
+    
+    # Initialize an empty list to store the data
+    data_list = []
```

Comment:
Having looked at this again I think it would be good to test this part and an easy way to do it would be to put the bit from line 136 onwards into a separate function called something like `make_generation_data` and then call that here instead. That function could take in the data_json object as an input and start and end time, this way it would be very easy to write some small unit tests for it.